src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -373,6 +374,18 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
     })
     await this.validate()
   }
+
+  /** Set validate on file and pay click. */
+  @Watch('getComponentValidate')
+  protected updateResolutionValidationDetail (): void {
+    // don't call validation during Jest tests because we are setting app valid
+    if (!this.isJestRunning) {
+      this.$refs.createSpecialResolutionRef.validate()
+      this.$refs.resolutionDatePickerRef.validateForm()
+      this.$refs.signatureDatePickerRef.validateForm()
+    }
+  }
```

Comment:
In this case, we can't use unit test files to test the validity of this form?

Reply:
This is to show validation error message with red color and red color label